lib/transition/import/dns_details.rb

```diff
@@ -18,14 +18,18 @@ def import!
         hosts.each do |host|
           $stderr.print '.'
 
-          if cname_record = record(host.hostname, Resolv::DNS::Resource::IN::CNAME)
+          cname_record = record(host.hostname, Resolv::DNS::Resource::IN::CNAME)
```

Comment:
It was complaining about assignment in the conditional as it's often a typo of / confused for equality.  I'll see if there's a different way to arrange this.